apps/pod-issuer-server/src/routes/gov.ts

```diff
@@ -0,0 +1,110 @@
+import express, { Request, Response } from "express";
+import { expressjwt, Request as JWTRequest } from "express-jwt";
+import jwt from "jsonwebtoken";
+import { POD, PODEntries } from "@pcd/pod";
+import { getGovUserByEmail, getIDPODByEmail, saveIDPOD } from "../stores/gov";
+
+const gov = express.Router();
+
+gov.post("/login", (req: Request, res: Response) => {
+  const inputs: { email: string; password: string } = req.body;
+  if (!inputs.email || !inputs.password) {
+    res.status(400).send("Missing query parameter");
+    return;
+  }
+
+  // In practice, get user information from database by email,
+  // Here for demo purposes, we'll allow any email with @zoo.com domain
+  if (!inputs.email.endsWith("@zoo.com")) {
+    res.status(401).send("Invalid email or password");
+    return;
+  }
+
+  // In practice, check if the encrypted password match
+  // This is just for demo purposes
+  if (!inputs.password.startsWith("zoo")) {
+    res.status(401).send("Invalid email or password");
+    return;
+  }
+
+  // Signing JWT, valid for 1 hour
+  const token = jwt.sign(
+    { email: inputs.email },
+    process.env.GOV_EDDSA_PRIVATE_KEY!,
+    { algorithm: "HS512", expiresIn: "1h" }
+  );
+  res.send(token);
+});
+
+gov.post(
```

Comment:
Several of my comments from the deel route also apply here, and I won't repeat them.